command/project_set.go

```diff
@@ -68,9 +79,20 @@ func (cmd *ProjectSet) DoRun(args []string) (err error) {
 		return HandleError(errors.New("Project not found, please check the project name. You can get a list of your projects by running `nerd project list`."))
 	}
 
-	var p populator.P
-	p = &populator.EnvPopulator{}
-	p.SetKubeConfigFile()
+	var kubeConfigFile string
+	if os.Getenv("KUBECONFIG") == "" {
```

Comment:
This can also be an optional command option that defaults to the env value "KUBECONFIG", see: https://godoc.org/github.com/jessevdk/go-flags and search for 'env'.  We also use it in the old 'commands/opts.go' file